backend/plugins/payment_api/src/apis/khanbank/api.ts

```
@@ -0,0 +1,132 @@
+      // const account = await sendCommonMessage('khanbank', {
+      //   subdomain: this.subdomain,
+      //   action: 'accountInfo',
+      //   data: { configId: this.configId, accountNumber: this.accountNumber },
+      //   isRPC: true,
+      //   defaultValue: null,
+      // });
+
+      // return { accountNumber: this.accountNumber, ibanAcctNo: this.ibanAcctNo, ...account };
+    } catch (e) {
+      return { error: e.message };
+    }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)